### ISSUE TITLE
Exemplar support

### ISSUE THREAD

**Issue by emschwartz** [autometrics-dev/autometrics-py#41]
Looks like this might be blocked waiting on upstream support:
- https://github.com/open-telemetry/opentelemetry-python/issues/2407
- https://github.com/prometheus/client_python/issues/648

**Comment by actualwitch** [comment]
[This PR](https://github.com/autometrics-dev/autometrics-py/pull/51) partially resolves the issue - for now exemplars are only supported in prometheus client, but for people who enable that backend for autometrics-py we expose a `AUTOMETRICS_EXEMPLARS` env var you can use to enable them in autometrics itself.

**Comment by emschwartz** [comment]
@actualwitch I think we can consider this done for now and potentially open another issue for later about supporting exemplars with OpenTelemetry whenever they add support for the feature upstream
